.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug
+about: "Report confirmed bugs. For unconfirmed bugs please visit our mailing list https://groups.google.com/forum/#!forum/janusgraph-users"
+labels: "kind/bug/possible"
+
+---
+
+Please post all questions and issues on https://groups.google.com/forum/#!forum/janusgraph-users
```

Comment:
Please consider hiding the URL behind the list name, i.e., `[janusgraph-users](https://groups.google.com/forum/#!forum/janusgraph-users)`.